articles/client-platforms/react/05-linking-accounts.md

```diff
@@ -0,0 +1,372 @@
+---
+title: Linking Accounts
+description: This tutorial will show you how to integrate Auth0 with ReactJS to link accounts.
+---
+
+<%= include('../../_includes/_github', {
+  link: 'https://github.com/auth0-samples/auth0-react-sample/tree/master/04-Linking-Accounts',
+}) %>_
+
+In some cases, there could be need for you to link multiple accounts. One very common situation is when a user signed up with email and password which provides very little information about the user. You can urge the user to link their account to an OAuth provider like Facebook or Google. In this article you'll see how to update the application created in [User profile](/quickstart/spa/react/03-user-profile) to allow the user to link or unlink other OAuth provides to his account.
+
+## 1. Show Linked Accounts Information
+
+User's profile contains an array of identities which is made of profile information from other providers. You can see this by accessing the [Auth0 users page](${uiURL}/#/users), selecting a user and scrolling down to the identities. This is what it looks like after linking Gmail:
+
+![User identities](/media/articles/users/user-identities-linked.png)
```

Comment:
@flaviogranero 
Please resize this image. Currently it is 1212 × 862 px. The max width for the images should be 750 px.
Thnx